carta/image.py

```diff
@@ -73,6 +73,9 @@ def new(cls, session, path, hdu, append, complex, expression):
         if complex:
             image_arithmetic = True
             file_name = f'{expression}("{file_name}")'
+        elif complex is False and file_name == "":
```

Comment:
I would prefer not to guess here, and to have explicit flags set by the functions in the session. I will make a more detailed suggestion in a comment for unifying the complex expression and arbitrary LEL expressions.